projects/plugins/jetpack/json-endpoints/class.wpcom-json-api-upload-media-v1-1-endpoint.php

```diff
@@ -105,6 +117,10 @@ public function callback( $path = '', $blog_id = 0 ) {
 			if ( count( $jetpack_media_files ) > 0 ) {
 				add_filter( 'upload_mimes', array( $this, 'allow_video_uploads' ) );
 
+				// get_space_used() checks blog upload directory storage,
+				// so filter it temporarily to return only video storage used.
+				add_filter( 'pre_get_space_used', 'videopress_filter_jetpack_get_space_used' );
```

Comment:
I think you misunderstood my comment earlier. We **do** need the `function_exists` check here, leaving it out will make it blow up when the filter is applied because the function doesn't exist.

What I was referring to in my comment earlier was regarding the `remove_filter()` call that still needs to be added. **That** doesn't need a `function_exists` guard.

Reply:
Ah apologies, added back.

Reply:
(turns out I was wrong, see other comment)